### DISCUSSION TITLE
Expected time to win in chia farm summary

### DISCUSSION THREAD

**Discussion by jayhohoho2019** [Chia-Network/chia-blockchain#9052] [Support]
Could someone clarify how this is calculated?  I thought it's just derived from a farm's total plots as a % of the total network space.   But I just noticed while "Estimated network space" went down (to below 34 EiB), this "Expected time to win:" actually went up by ~ 30%, even though "Plot count for all harvesters" has remained exactly the same.   All the labels and values are from the output of "chia farm summary"

**Comment by jayhohoho2019** [comment]
Now "Expected time to win" has increased by another 25% while "Estimated network space" has dropped below 33 EiB (from below 34 EiB).  "Plot count for all harvesters" has remained exactly the same.

**Reply by jayhohoho2019** [reply]
During the last two hours,  19:37pm - 21:46pm EDT, the number of WARNING lines in my farmer log that contains the "add_spendbundle" has increased by 40.  The time reported in each line is around 2.5 seconds.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
